plugins/rp-pico/fu-rp-pico-device.c

```diff
@@ -47,6 +47,8 @@ fu_rp_pico_device_detach(FuDevice *device, FuProgress *progress, GError **error)
 					    &error_local)) {
 		if (g_error_matches(error_local, FWUPD_ERROR, FWUPD_ERROR_READ)) {
 			g_debug("ignoring expected error %s", error_local->message);
+		} else if (g_error_matches(error_local, FWUPD_ERROR, FWUPD_ERROR_NOT_FOUND)) {
```

Comment:
Can you combine these into one if please; then LGTM.

Reply:
Done